templates/zerver/help/import-from-slack.md

```diff
@@ -221,100 +165,16 @@ token from being used to access your Slack workspace in the future.
 
 ## Get your organization started with Zulip
 
-Once the import process is completed, you will need to:
-
-{start_tabs}
-
-1. [Configure the settings for your organization](/help/customize-organization-settings),
-   which are not exported from Slack. This includes settings like [email
-   visibility](/help/restrict-visibility-of-email-addresses), [message editing
-   permissions](/help/configure-message-editing-and-deletion#configure-message-editing-and-deletion_1),
-   and [how users can join your organization](/help/restrict-account-creation).
-
-2. All users from your Slack workspace will have accounts in your new Zulip
-   organization. However, you will need to let users know about their new
-   accounts, and [decide how they will log
-   in](/help/import-from-slack#decide-how-users-will-log-in) for the first time.
-
-3. Share the URL for your new Zulip organization, and (recommended) the [Getting
-   started with Zulip guide](/help/getting-started-with-zulip).
-
-4. Migrate any [integrations](/integrations), which is easy to do with Zulip's
-   [Slack-compatible incoming webhook](/integrations/doc/slack_incoming).
-
-{end_tabs}
-
-
-### Decide how users will log in
-
-When user accounts are imported from Slack, users initially do not have passwords
-configured. There are a few options for how users can log in for the first time.
+{!import-get-your-organization-started.md!}
 
 !!! tip ""
-    For security reasons, Slack passwords are never exported.
```

Comment:
We should not drop this tip! But we can drop "Slack", if that helps:

> For security reasons, passwords are never exported.

Reply:
I moved this bit of info to the first line of the paragraph above it, but dropping "Slack" and leaving it as a separate tip works for me too.